TSOMessageHub/XML/AfrrSignal.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+namespace TSOMessageHub.XML
```

Comment:
consider making it file-scoped, (aka `namespace TSOMessageHub.XML;`)

Reply:
fixed